backend/src/packages/sections/section.service.ts

```diff
@@ -205,6 +212,37 @@ class SectionService
 
     return portfolioContent;
   }
+
+  private async createServiceContent(
+    prompt: string,
+  ): Promise<SiteServiceContent> {
+    const cardsContent = await Promise.all(
+      Array.from({ length: SectionService.serviceCardsQuantity }, () =>
+        openAI.createCompletion(prompt),
+      ),
+    );
+
+    const cards = cardsContent.map((cardContent) => ({
+      title: cardContent['title'] ?? '',
+      picture: cardContent['pictureDescription'] ?? '',
+      description: cardContent['description'] ?? '',
+    }));
+
+    await initAsyncItemsQueue(cards, async (card) => {
+      const image = await openAI.createImage(
+        `Flat icon for site with white background. Icon image colors: black and peach  ${card.picture}`,
+      );
+
+      const { url } = await this.file.upload({ file: image });
+
+      card.picture = url;
+    });
+
+    return {
+      title: 'Our Services',
```

Comment:
```suggestion
```
you can hardcode it on the frontend

Reply:
in section.service.ts we create full object for every part of site and send it to the front, even if title is hardcoded. It will look weird when titles for Portfolio and Feedback are hardcoded on the back but for service on the front. And if in future there will be need to create title by AI...